Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+ADD src/quarto.py /
```

Comment:
I would instead recommend mounting it as a volume (option `-v` in your Docker command line)